docs/readmes/orc8r/deploy_install.md

```diff
@@ -23,6 +26,9 @@ If your AWS account is not blank, this can cause errors while Terraforming.
 If you know what you're doing, this is fine - otherwise, consider signing up
 for a new account.
 
+Finally, our install process assumes the chosen region contains at least 3
```

Comment:
Can we perhaps link some url showing the number of availability zones per region